src/main/java/com/groupseven/pdfproject/model/Action.java

```diff
@@ -14,34 +14,24 @@
  */
 public interface Action {
 
-    /**
-     * Performs the action
-     */
+    /// \Performs the action
     void execute();
 
-    /**
-     * Sets up the environment for the action to take place
-     * 
-     * @param event
-     *            Event triggering the action
-     * 
-     * @return a constructed action based on the Event source
-     */
+    /// \Sets up the environment for the action to take place
```

Comment:
The `\` character doesn't need to be on every line. The `@` symbol and `\` can be used interchangeably to signify a special directive for doxygen like `\brief` will put the rest of the line in a short brief about the method.

this is what the doxygen being generated by this looks like:

![image](https://user-images.githubusercontent.com/18319310/116197217-75058000-a702-11eb-94da-3c546c78b965.png)

You should update these comments to only include the necessary `\` characters.

Reply:
Commenting has been remedied so doxygen is displayed properly